### PR TITLE
make fonts changeable. Fixes #10

### LINKED ISSUE #10: How to change the font?

### PR DESCRIPTION
**Pull request by migmedia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +79,26 @@ See the default value [here](./templates/_variables.html)
...
+@import "../themes/juice/sass/_ultility.scss";
+@import "_text.scss";
+@import "../themes/juice/sass/_markdown.scss";
+...
+```
+and then load the fonts in the `templates/index.html`.
+```html
+{% extends "juice/templates/index.html" %}
+{% block fonts %}
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
```
Comment (Folyd) [inline README.md]: I suggest removing this link to keep the example as simple as possible.

File: README.md [hunk 2]
```diff
@@ -78,6 +79,26 @@ See the default value [here](./templates/_variables.html)
...
+### Fonts
+Copy the `juice.scss` and `_text.scss` files from `template/juice/sass/` to local `sass/`-dir. And change the 
+`@import`-lines.
+
+```scss
+@import "../themes/juice/sass/_ultility.scss";
+@import "_text.scss";
+@import "../themes/juice/sass/_markdown.scss";
+...
+```
```
Comment (Folyd) [inline README.md]: I didn't get why we need to copy those scss files

**Review by Folyd — COMMENTED** [review]
LGTM, just a little nit-picks,  thanks for working on this.

File: README.md (hunk 2 above)
Comment (migmedia) [reply README.md]: I changed the font-selection to variables. So it's changeable in the `_variables.html` file.

File: README.md (hunk 1 above)
Comment (migmedia) [reply README.md]: I have shortened it, but I want to show an alternative to `fonts.google.com`.

**Review by Folyd — APPROVED** [review]
LGTM, thanks

**Pull request merged**